metrics/web/lib/common/presentation/widgets/metrics_dialog.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+
+/// The widget that represents metrics dialog.
+class MetricsDialog extends StatelessWidget {
+  final Color backgroundColor;
+  final double elevation;
+  final ShapeBorder shape;
+  final EdgeInsetsGeometry padding;
+  final double maxWidth;
+
+  final Widget title;
+  final EdgeInsetsGeometry titlePadding;
+
+  final Widget content;
+  final EdgeInsetsGeometry contentPadding;
+
+  final List<Widget> actions;
+  final EdgeInsetsGeometry actionsPadding;
+  final MainAxisAlignment actionsAlignment;
+
+  /// Creates an [MetricsDialog].
+  ///
+  /// [backgroundColor] is a background color of the surface of this [MetricsDialog].
+  /// [elevation] is an elevation of this [MetricsDialog].
+  /// [shape] is a shape of this [MetricsDialog] border.
+  /// [padding] is a padding around the [MetricsDialog] child.
+  /// [maxWidth] is a maximum available width of this [MetricsDialog].
+  /// [title] is a title of this [MetricsDialog].
+  /// [titlePadding] is a padding around the title.
+  /// [content] is a content of this [MetricsDialog].
+  /// [contentPadding] is a padding around the content.
+  /// [actions] is a set of actions that are displayed at the bottom of this [MetricsDialog].
+  /// [actionsPadding] is a padding around the actions.
+  /// [actionsAlignment] is a main axis alignment of the [MetricsDialog]'s actions.
```

Comment:
See suggestion here: https://dart.dev/guides/language/effective-dart/documentation#do-use-prose-to-explain-parameters-return-values-and-exceptions